### PR TITLE
Added an indentation style option for class generation, and user-defined tabs.

### PR DESCRIPTION
**Pull request by Gr1moire**
Hi !

Since there is no Epitech-specific coding style for C++ anymore, I did some small changes enabling user's to select an indentation style for access specifiers when generating classes. By default, "public", "protected" and "private" keywords will have an indentation of 1, but users can now choose to not indent them. They will be prompted with a new question about it while configuring the extension, the answer is then stored in Config.indentAccessSpecifiers.

I also fixed tabs while generating classes : most vscode's users at Epitech have 4-spaces tabs, not \t. Classes generated by the extension were always indented with a \t and it wasn't possible to configure it without changing the code itself. It was kind of strange to have indentation made of \t in one place, and spaces in some other place.
Tabs used for class generation are now defined by the editor's settings (editor.insertSpaces and editor.tabSize) so that everything in the code is indented the same way.

Hope this helps! :)

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -108,7 +108,8 @@ export function activate(context: vscode.ExtensionContext) {
                     editContent = appendIfndef(editContent, name, fileInfo, config);
                 }
                 if (config.autoGenerateClasses && fileInfo.langId == "cpp" && isUpper(className[0])) {
-                    editContent = appendClass(editContent, className, fileInfo);
+                    console.log(config.indentAccessSpecifiers);
```
Comment (Hirevo) [inline src/extension.ts]: I think we may remove that `console.log(...)`.

File: src/generators.ts [hunk 2]
```diff
@@ -4,6 +4,8 @@ import * as vscode from "vscode";
 import * as escapeRegExpString from "escape-string-regexp";
 import { DAYS, EOLS, MONTHS, SUPPORTED_LANGUAGES, SYNTAX } from "./constants";
 import { Config, FileInfo, HeaderGenerator } from "./interfaces";
+import { configureSettings } from "./config";
+import { stringLiteral } from "babel-types";
```
Comment (Hirevo) [inline src/generators.ts]: These imports seem to be unused.

**Comment by Gr1moire** [comment]
Hi again,

My bad, I forgot to remove these lines after debugging. It's now done! :)

**Comment by Hirevo** [comment]
**v1.14.0** is now released with these features, with all due credits in the changelogs.  
Many thanks, again, for the contribution !

**Pull request merged**